cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -44,13 +44,16 @@ import (
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/master/ports"
+
 	// add the kubernetes feature gates
```

Comment:
IMO, we have given the comment in the next line, this blank line seems no need

Reply:
Fixed